### DISCUSSION TITLE
CLI version for scripting

### DISCUSSION THREAD

**Discussion by jimlynnjulian** [dnGrep/dnGrep#845] [General]
Hello,

Can the app be run from the command line? I need a grep that reads unicode and can be used in a script.

Thank you,
Jim J.

**Comment by doug24** [comment]
Yes, dnGrep can be run from a command line, but it mainly is a GUI application.  Here is the [documentation](https://github.com/dnGrep/dnGrep/wiki/Command-Line).  If you don't want to see the application window, you can set up and run a search, output a file with the search results and exit from the command line arguments.

If you want a command-line only program that outputs to the shell, I suggest trying [ripgrep](https://github.com/BurntSushi/ripgrep).
